src/extension.ts

```diff
@@ -240,13 +240,16 @@ export class Extension {
       const workspaceFolderPath = workspaceFolder.uri.fsPath;
       if (configFilePath.includes('test-results') && !workspaceFolderPath.includes('test-results'))
         continue;
-      const playwrightInfo = await this._playwrightTest.getPlaywrightInfo(workspaceFolderPath, configFilePath);
-      if (!playwrightInfo) {
+      let playwrightInfo = null;
+      try {
+        playwrightInfo = await this._playwrightTest.getPlaywrightInfo(workspaceFolderPath, configFilePath);
+      } catch (error) {
         if (showWarnings) {
           this._vscode.window.showWarningMessage(
-              this._vscode.l10n.t('Please install Playwright Test via running `npm i --save-dev @playwright/test`')
+              error instanceof NodeJSNotFoundError ? error.message : this._vscode.l10n.t('Please install Playwright Test via running `npm i --save-dev @playwright/test`')
           );
         }
+        console.error('[Playwright Test]:', (error as any)?.message, 1);
```

Comment:
I like to keep this error which we consider fatal in DevTools, like we had it before. If the user did e.g. disable all the warnings from the extension, what he can, this would not be shown and hidden everywhere.